packages/animated/package.json

```diff
@@ -30,15 +30,42 @@
   },
   "sideEffects": false,
   "dependencies": {
-    "@pixi/react": "*"
+    "@pixi/react": "*",
+    "@pixi/app": "^7.1.1",
+    "@pixi/core": "^7.1.1",
+    "@pixi/display": "^7.1.1",
+    "@pixi/graphics": "^7.1.1",
+    "@pixi/math": "^7.1.1",
+    "@pixi/mesh-extras": "^7.1.1",
+    "@pixi/text": "^7.1.1",
+    "@pixi/text-bitmap": "^7.1.1",
+    "@pixi/ticker": "^7.1.1",
+    "@pixi/sprite": "^7.1.1",
+    "@pixi/sprite-tiling": "^7.1.1",
+    "@pixi/sprite-animated": "^7.1.1",
+    "@pixi/particle-container": "^7.1.1"
```

Comment:
Could be 😆 - sometimes I just don't know - these are only here for the typings I think, so I guess that is dev?

Reply:
That doesn't actually make a difference does it? I think as long as they're passed as `external` to `rollup` it doesn't matter - but I think it makes sense semantically

Reply:
It does make a difference because when you `npm i @pixi/animated` it would pull in `7.1.1` of pixi

rollup may exclude it from the build but npm will still install these modules, which could lead to multiple versions of pixi

Reply:
Resolved in dbfc3b440aa01fd64b015d7516f989ec190e41c9